projects/plugins/jetpack/modules/copy-post.php

```diff
@@ -317,10 +317,12 @@ public function add_row_action( $actions, $post ) {
 		);
 		$edit_action = array(
 			'jetpack-copy' => sprintf(
-				'<a href="%s" aria-label="%s">%s</a>',
+				'<a href="%s" aria-label="%s">%s <img src="%s" alt="%s" width="16" height="16" style="vertical-align:text-bottom"></a>',
 				esc_url( $edit_url ),
-				esc_attr__( 'Copy this post.', 'jetpack' ),
-				esc_html__( 'Copy', 'jetpack' )
+				esc_attr__( 'Copy this post with Jetpack', 'jetpack' ),
+				esc_html__( 'Copy', 'jetpack' ),
+				plugins_url( '', JETPACK__PLUGIN_FILE ) . '/images/jetpack-logomark-green.svg',
+				esc_attr__( 'Jetpack', 'jetpack' )
```

Comment:
Since the image is decorative, I do not think we would need an alt attribute. I do not think it needs to be specified to screen readers.

Reply:
Since the link has an `aria-label` attribute, I believe screen readers will ignore the inner content (I tested with VoicOver and NVDA). An empty `alt` attribute would be perfectly fine.

Reply:
Removed 👍